server/service/src/invoice_line/outbound_shipment_line/delete/mod.rs

```diff
@@ -143,21 +147,81 @@ mod test {
             Err(ServiceError::CannotEditInvoice)
         );
 
-        //TODO DatabaseError, NotThisStoreInvoice
+        // NotThisStoreInvoice
+        assert_eq!(
+            service.delete_outbound_shipment_line(
+                &context,
+                &mock_store_a().id,
+                DeleteOutboundShipmentLine {
+                    id: mock_outbound_shipment_c_invoice_lines()[0].id.clone()
+                }
+            ),
+            Err(ServiceError::NotThisStoreInvoice)
+        );
+
+        //TODO DatabaseError
```

Comment:
I think we can't test for db errors. This would mean something is really wrong and it would be difficult to simulate it. I think all other errors that could happen in normal operation have an enum variant.